sdks/java/extensions/google-cloud-platform-core/src/test/java/org/apache/beam/sdk/util/FastNanoClockAndSleeper.java

```diff
@@ -15,7 +15,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.beam.sdk.testing;
```

Comment:
We are working on moving stuff out of org.apache.beam.sdk.util 
Most of what is left is GCS related, @dhalperi do you have a PR that does this?